src/Nox.Types/Types/EncryptedText/TypeOptions/EncryptedTextTypeOptions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Security.Cryptography;
+
+namespace Nox.Types;
+
+public class EncryptedTextTypeOptions
+{
+    public EncryptionAlgorithm EncryptionAlgorithm { get; set; } = EncryptionAlgorithm.Aes;
+
+    /// <summary>
+    /// The secret key to use for the algorithm.
+    /// </summary>
+    public byte[]? PublicKey { get; set; }
```

Comment:
I'm not sure how the user can set this on the schema... maybe it needs to be string in a Base64 encoding?